src/base/_variables.scss

```diff
@@ -16,6 +16,7 @@ $color-sage: #a8c3be;
 $color-sagelt: #d7e5e2;
 $color-dove: #f5f3f2;
 $color-greymd: #87888e;
+$color-greydarker: #6e7379;
```

Comment:
Thoughts on using Color-black here? #53555e and #6e7379 are really close, but I'll defer to you :)